src/index.html

```diff
@@ -1,24 +1,29 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="IE=edge">
-    <title>Firefox Lightbeam</title>
-    <meta name="description" content="Firefox Lightbeam is a Firefox add-on that uses interactive visualizations to show you the relationships between third parties and the sites you visit.">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
-    <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
-    <link rel="stylesheet" href="css/style.css" type="text/css">
-    <script src="js/storeChild.js" type="text/javascript"></script>
-    <script src="ext-libs/d3.min.js"></script>
-    <script src="ext-libs/dialog-polyfill.js"></script>
-    <script src="js/viz.js" type="text/javascript"></script>
-    <script src="js/lightbeam.js" type="text/javascript"></script>
-  </head>
-  <body>
-    <aside class="max-graph">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="IE=edge">
+  <title>Firefox Lightbeam</title>
+  <meta name="description"
+    content="Firefox Lightbeam is a Firefox add-on that uses interactive visualizations to show you the relationships between third parties and the sites you visit.">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
+  <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
+  <link rel="stylesheet" href="css/style.css" type="text/css">
+  <script src="js/greencheck.js" type="text/javascript"></script>
```

Comment:
I don't think `greencheck.js` is needed in the codebase now, so it's worth taking out.

I couldn't find the file at any rate, and it might make sense to have a separate module eventually, but it shows up as a failed request at present in the console.

Reply:
Eagle eye, thank you! Yes, that was from a very early version and I ended up keeping it all in the same file instead of making a second one. I've removed this from the index.html file